### PR TITLE
Added changes for note deletion page UI #2

### PR DESCRIPTION
**Pull request by nivetha-yadhavan**
image
image

### CODE REVIEW

**Comment by yedhink** [comment]
@nivetha-yadhavan Please go through the following articles:
- https://handbook.neetokb.com/articles/git-and-github-essentials
- https://handbook.neetokb.com/articles/understanding-git-rebase

Importantly, at BigBinary, we always use `Squash and Merge` rather than fast forward merge like you've done with this PR. Refer: https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 import EmptyNotesListImage from "images/EmptyNotesList";
-import { Button } from "neetoui";
+import { Button, Toastr } from "neetoui";
 import { Container, Header } from "neetoui/layouts";
 
 import EmptyState from "components/Common/EmptyState";
 
-import { NOTES as notesList } from "./constants";
+import { NOTES as notes } from "./constants";
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:9]: No need do this aliasing. Let literal constants be named as a constant itself.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 2]
```diff
@@ -17,6 +17,19 @@ const Notes = () => {
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [showMenu, setShowMenu] = useState(false);
+  const [notesList, setNotes] = useState(notes);
+  const [selectedNoteId, setSelectedNoteId] = useState();
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:21]: This is a bad practice. Don't declare states without providing a default value signifying what kind of data the state can hold.

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 3]
```diff
@@ -17,6 +17,19 @@ const Notes = () => {
   const [showDeleteAlert, setShowDeleteAlert] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
   const [showMenu, setShowMenu] = useState(false);
+  const [notesList, setNotes] = useState(notes);
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:20]: This is again a bad practice. State name and setter name should be consistent. It should always be:
```
state, setState
```

So here:
```suggestion
const [notes, setNotes] = useState(NOTES);
```

File: app/javascript/src/components/Dashboard/Notes/index.jsx [hunk 4]
```diff
@@ -17,6 +17,19 @@ const Notes = () => {
...
+  const [selectedNoteId, setSelectedNoteId] = useState();
+
+  const handleDelete = () => {
+    try {
+      const notes = notesList.filter(c => c.id !== selectedNoteId);
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/index.jsx:25]: Don't use random variable names like `c` etc which doesn't make any sense in the context. Here you are iterating through the notes. Thus each item is a `note`. NOT a `c`!

**Review by yedhink — COMMENTED** [review]
@nivetha-yadhavan Few comments added.

**Comment by nivetha-yadhavan** [comment]
@yedhink addressed the review comments in https://github.com/nivetha-yadhavan/wheel/pull/9

**Pull request merged**